src/Raven.Client/Documents/Operations/Attachments/LimitedStream.cs

```diff
@@ -29,9 +29,16 @@ public override void Flush()
             throw new NotSupportedException();
         }
 
-        public override long Seek(long offset, SeekOrigin origin)
+        public override long Seek(long position, SeekOrigin origin)
```

Comment:
Can we add a test for that in LimitedStreamTests ?

Reply:
done